providers/apollo/write.go

```diff
@@ -0,0 +1,63 @@
+package apollo
+
+import (
+	"context"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/amp-labs/connectors/common/naming"
+	"github.com/spyzhov/ajson"
+)
+
+// Write creates/updates records in apolllo.
+func (c *Connector) Write(ctx context.Context, config common.WriteParams) (*common.WriteResult, error) {
+	var write common.WriteMethod
+
+	url, err := c.getAPIURL(config.ObjectName)
+	if err != nil {
+		return nil, err
+	}
+
+	// sets post as default
+	write = c.Client.Post
+
+	// prepares the updating data request.
+	if len(config.RecordId) > 0 {
+		url = url.AddPath(config.RecordId)
```

Comment:
Can you please add screenshots of both record creates and updates in the PR description?